### DISCUSSION TITLE
kallisto index

### DISCUSSION THREAD

**Discussion by laurel-nave-powers** [sr320/course-fish546-2021#124] [Q&A] [answered]
Hello, I have been trying to use kallisto to do an alignment but am stuck at the index phase. It isn't giving me an index but it also isn't giving me any errors. It seems to stop half way through the process. 
Here is the notebook I'm working in: https://github.com/fish546-2021/Laurel-genes/blob/main/code/04_kallisto.ipynb

**Comment by sr320** [comment]
It might just take a long time- how do you know it "seems to stop"?

**Reply by laurel-nave-powers** [reply]
That's what I thought too, but the cell loses the asterix next to it and changes to a number, doesn't that indicate that it is done running?

**Reply by sr320** [reply]
See below. it should work super fast, seconds..

**Reply by sr320** [reply]
and yes you are correct , loss of asterisk and number change indicates complete

**Comment by sr320** [comment, marked as answer]
Worked for me. See notebook 04.1 in your repo. 
I dowloaded the fa file and placed in `data`, created index there too. Both files should be in your repo as pushing finishes

**Reply by laurel-nave-powers** [reply]
Interesting I wonder why it wasn't working for me... thank you for your help!

**Reply by sr320** [reply]
Yep - I see no issues with your code.. you could try again and see if you get it.
